tests/src/test/java/alluxio/underfs/sleepfs/SleepingUnderFileSystemOptions.java

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
if it was auto-generated, then perhaps even the unit tests could be auto-generated ... alas neither is

the reason I ask for a unit test is because in the past, unit tests for the option classes have caught typos where setter and / or getter method would be setting / getting the value of a different field ... such issues should be caught by a unit tests because if they are not, they either lead to a goose chase debugging a test that expect the functionality to work, or worse go undetected